client/src/main/java/TestService.java

```diff
@@ -0,0 +1,65 @@
+import java.io.BufferedReader;
+import java.io.IOException;
+import java.io.InputStreamReader;
+import java.io.PrintWriter;
+import java.net.ServerSocket;
+import java.net.Socket;
+import java.nio.charset.StandardCharsets;
+
+public class TestService {
```

Comment:
I'd rather delete this file as it'll be completely different in the next couple of commits.